src/containers/shared/components/Transaction/XChainAddAttestation/AccountCreateAttestation.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import Account from '../../Account'
+import { Amount } from '../../Amount'
+import { SimpleRow } from '../SimpleRow'
+
+export interface AccountCreateAttestationProps {
+  send: string
+  account: string
+  destination: string
+  signature: string
+}
+
+export const AccountCreateAttestation = (
+  props: AccountCreateAttestationProps,
+) => {
+  const { t } = useTranslation()
+  const { send, account, destination, signature } = props
+
+  return (
+    <div className="claim-attestation" key={signature}>
+      <div className="claim-attestation-title">
+        XChain Create Account Attestation Batch Element
+      </div>
+      <SimpleRow label={t('send')} data-test="send">
```

Comment:
What does `data-test` do?

Reply:
It serves a selector that can be used in tests to assert the values of a component.  No different than adding a className except there are tools that will strip out data-test instances from a prod build.  We don't do that just yet.  It also serves as something not style or functionally related solely there for tests